CHANGELOG.md

```diff
@@ -5,6 +5,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 
 ## [Unreleased]
+### Removed
+- `OrderedDict`s in favour of default dictionaries. Python>=3.7 now required.
```

Comment:
```suggestion
### Changed
- Python>=3.7 now required.
```
We essentially don't need to specify the internal change (OrderedDict -> dict), but just the external one (Python dep)

Reply:
Adding to that, "OrderedDict" is not a feature of qrm, just a detail of how we implemented some features.